src/utils/content.ts

```
@@ -0,0 +1,73 @@
+    logger.error('Error detecting content type:', error);
+    return undefined;
+  }
+};
+
+export const getDataType = async (
+  content: Buffer | string,
+): Promise<{
+  contentType: 'text' | 'audio' | 'image' | 'video' | 'sticker' | 'document';
+  mimeType: string;
+}> => {
+  if (typeof content === 'string') content = Buffer.from(content);
```

Biome (1.9.4)

[error] Reassigning a function parameter is confusing.

The parameter is declared here:

Use a local variable instead.

(lint/style/noParameterAssign)